src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -2211,9 +2215,9 @@ class Program
 
             var comp = CreateCompilationWithMscorlibAndSystemCore(text, parseOptions: TestOptions.Regular7_1);
             comp.VerifyDiagnostics(
-                // (6,47): error CS0845: An expression tree lambda may not contain a coalescing operator with a null or default literal left-hand side
```

Comment:
>// (6,47): error CS0845: An expression tree lambda may not contain a coalescing operator with a null or default literal left-hand side [](start = 16, length = 134)

Can we remove this error and the code that reports it?

Reply:
This error is still reported. For `Expression<> e = () => null ?? x;` for example.

---
In reply to: [135880750](https://github.com/dotnet/roslyn/pull/21757#discussion_r135880750) [](ancestors = 135880750)